StyleCop.Analyzers/StyleCop.Analyzers/Lightup/MemberDeclarationSyntaxExtensions.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace StyleCop.Analyzers.Lightup
+{
+    using System;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+    internal static class MemberDeclarationSyntaxExtensions
```

Comment:
📝 These extensions should fall back to support derived types that offered these properties prior to moving them to `MemberDeclarationSyntax`

Reply:
➡️ #3258